modules/Bio/EnsEMBL/Compara/PipeConfig/DumpAllForRelease_conf.pm

```diff
@@ -95,7 +95,7 @@ sub default_options {
         'dump_dir'        => '#dump_root#/release-#curr_release#',
 		'lastz_dump_path' => 'maf/ensembl-compara/pairwise_alignments', # where, from the FTP root, is the LASTZ dumps?
         'reuse_prev_rel'  => 1, # copy symlinks from previous release dumps
-        #'updated_mlss_ids' => [1142,1143,1134,1141], #the list of mlss_ids that we have re_ran/updated and cannot be detected through first_release
```

Comment:
Can you leave this line as reminder it is an array of integers what is expected?